resources/profiles/Anker.ini

```diff
@@ -26,156 +26,817 @@ default_materials = Generic PLA+ @ANKER; Generic PLA @ANKER; Generic PET @ANKER;
+    standby_temperature_delta = -5
+    support_material_auto = 0
+    support_material = 0
+    support_material_angle = 0
+    support_material_buildplate_only = 0
+    support_material_contact_distance = 0.1
+    support_material_enforce_layers = 0
+    support_material_extruder = 0
+    support_material_interface_contact_loops = 0
+    support_material_interface_extruder = 0
+    support_material_interface_layers = 2
+    support_material_interface_spacing = 0.2
+    support_material_interface_speed = 80%
+    support_material_pattern = rectilinear
+    support_material_spacing = 2
+    support_material_speed = 125
+    support_material_synchronize_layers = 0
+    support_material_threshold = 55
+    support_material_with_sheath = 0
+    support_material_xy_spacing = 50%
+    thick_bridges = 0
+    thin_walls = 0
+    top_solid_infill_speed = 150
+    top_infill_extrusion_width = 0.4
+    top_fill_pattern = rectilinear
+    bottom_fill_pattern = rectilinear
+    travel_speed = 250
+    travel_speed_z = 0
+    wipe_tower = 0
+    wipe_tower_bridging = 10
+    wipe_tower_rotation_angle = 0
+    wipe_tower_width = 60
+    wipe_tower_x = 170
+    wipe_tower_y = 140
+    xy_size_compensation = 0
 
 [print:*0.10mm*]
-inherits = *common*
-layer_height = 0.10
-first_layer_height = 0.10
-bottom_solid_layers = 7
-top_solid_layers = 9
-bridge_flow_ratio = 1
+    inherits = *common*
+    layer_height = 0.10
+    first_layer_height = 0.10
+    bottom_solid_layers = 7
+    top_solid_layers = 9
+    bridge_flow_ratio = 1
 
 [print:*0.20mm*]
-inherits = *common*
-layer_height = 0.20
-first_layer_height = 0.14
-bottom_solid_layers = 4
-top_solid_layers = 5
+    inherits = *common*
+    layer_height = 0.20
+    first_layer_height = 0.14
+    bottom_solid_layers = 4
+    top_solid_layers = 5
 
 [print:*0.30mm*]
-inherits = *common*
-layer_height = 0.30
-first_layer_height = 0.21
-bottom_solid_layers = 3
-top_solid_layers = 4
+    inherits = *common*
+    layer_height = 0.30
+    first_layer_height = 0.21
+    bottom_solid_layers = 3
+    top_solid_layers = 4
 
 
 [print:0.10 mm HIGHDETAIL (0.4 mm nozzle) @ANKER]
-inherits = *0.10mm*
-compatible_printers_condition = printer_model=~/(M5).*/ and nozzle_diameter[0]==0.4
+    inherits = *0.10mm*
+    compatible_printers_condition = printer_notes=~/.*(MACHINE_M5_).*/ and nozzle_diameter[0]==0.4
 
 [print:0.20 mm NORMAL (0.4 mm nozzle) @ANKER]
-inherits = *0.20mm*
-compatible_printers_condition = printer_model=~/(M5).*/ and nozzle_diameter[0]==0.4
+    inherits = *0.20mm*
+    compatible_printers_condition = printer_notes=~/.*(MACHINE_M5_).*/ and nozzle_diameter[0]==0.4
 
 
 [print:0.30 mm SUPERDRAFT (0.4 mm nozzle) @ANKER]
-inherits = *0.30mm*
-compatible_printers_condition = printer_model=~/(M5).*/ and nozzle_diameter[0]==0.4
+    inherits = *0.30mm*
+    compatible_printers_condition = printer_notes=~/.*(MACHINE_M5_).*/ and nozzle_diameter[0]==0.4
+
+[print:Precision - M5C (0.4 mm nozzle) @ANKER]
```

Comment:
This should inherit common and only change the values needed for this mode

Reply:
it means that it inherits *0.30mm*，and *0.30mm* inherits common

Reply:
you are correct. I misread this one.